context/app/static/js/components/SampleDetail.jsx

```diff
@@ -27,7 +27,7 @@ function SummaryData(props) {
     <>
       {organ && organ.length
         ? (<SummaryItem>{organ}</SummaryItem>) : null}
-      {origin_sample.organ && origin_sample.organ.length
+      {origin_sample.organ && origin_sample.organ.length && !organ
         ? (<SummaryItem>{origin_sample.organ}</SummaryItem>) : null}
```

Comment:
(With all of this, do we worry about details now, or, more likely, come back later and try to have something more uniform to work against?)

- Consider pulling the complicated logic out of the JSX and defining an `organ_name` above.
- For parallelism, should it be `organ && organ.length && !origin_sample.organ` for the first case? Or there are both, does the first take precedence?

Reply:
Will revisit, thanks!